python/understack-workflows/understack_workflows/nautobot.py

```diff
@@ -120,6 +127,22 @@ def update_cf(self, device_id: UUID, field_name: str, field_value: str):
         self.logger.info(f"save result: {response}")
         return response
 
+    def update_switch_interface_status(self, server_interface_id: UUID, new_status: str) -> NautobotInterface:
+        server_interface = self.interface_by_id(server_interface_id)
+        connected_endpoint = server_interface.connected_endpoint
+        if not connected_endpoint:
+            raise Exception("Interface {server_interface_id} not connected in Nautobot")
```

Comment:
missing f-prefix without which the string interpolation would not work

```suggestion
            raise Exception(f"Interface {server_interface_id} not connected in Nautobot")
```